src/Hydra/JsonSchema/SchemaFactory.php

```diff
@@ -74,24 +74,29 @@ public function buildSchema(string $resourceClass, string $format = 'jsonld', bo
                 ],
                 'hydra:totalItems' => [
                     'type' => 'integer',
-                    'minimum' => 1,
+                    'minimum' => 0,
                 ],
                 'hydra:view' => [
                     'type' => 'object',
                     'properties' => [
-                        '@id' => ['type' => 'string'],
-                        '@type' => ['type' => 'string'],
+                        '@id' => [
+                            'type' => 'string',
+                            'format' => 'iri-reference',
+                        ],
+                        '@type' => [
+                            'type' => 'string',
+                        ],
                         'hydra:first' => [
-                            'type' => 'integer',
-                            'minimum' => 1,
+                            'type' => 'string',
+                            'format' => 'iri-reference',
                         ],
                         'hydra:last' => [
-                            'type' => 'integer',
-                            'minimum' => 1,
+                            'type' => 'string',
+                            'format' => 'iri-reference',
                         ],
                         'hydra:next' => [
-                            'type' => 'integer',
-                            'minimum' => 1,
+                            'type' => 'string',
+                            'format' => 'iri-reference',
```

Comment:
I wasn't aware of the existence of `iri-reference`! We should use it everywhere we know that an IRI will be generated!

Reply:
@dunglas Do you have any in mind?

Reply:
he means in the rest of the core, so def another pr!